src/utils/index.ts

```diff
@@ -2,3 +2,4 @@ export * from './assertion.util';
 export * from './report.util';
 export * from './test-identification.util';
 export * from './screenshot.util';
+export * from './traceability.util';
```

Comment:
Picky comment, but to faciliate identifying if an item is already present in the barrel, it is helpful/a good practice to sort them alphabetically.

Reply:
Sorted items alphabetically